include/vara/Feature/FeatureModel.h

```diff
@@ -191,6 +175,26 @@ struct DOTGraphTraits<vara::feature::FeatureModel *>
     return S.str();
   }
 
+  static void buildCluster(llvm::raw_ostream &O, vara::feature::Feature *Node,
+                           int L = 1) {
```

Comment:
what is `L` could you please add a comment for the function that also describes the parameters